ProjFS.Mac/PrjFSKext/public/PrjFSCommon.h

```diff
@@ -45,6 +47,8 @@ enum PrjFSServiceUserClientType
 #define KEXT_STATIC_INLINE static inline
 #endif
 
+#ifdef __cplusplus
```

Comment:
Curious why we added this

Reply:
C and Objective-C don't support namespaces, so if you `#include` this header from a .m or .c file, you'll get syntax errors on that part.